### ISSUE TITLE
[feature-request] list who you're currently following in your profile

### ISSUE THREAD

**Issue by barathrm** [hummingbard/hummingbard#7]
What the title says.

**Comment by ChurchoftheSubGenius** [comment]
This should probably be toggle-able via settings.

**Comment by barathrm** [comment]
> This should probably be toggle-able via settings.

Yeah, I should've been more clear. I want to see for myself who I'm following, not necessarily to allow others to know who I'm following.
